src/snowflake/cli/plugins/nativeapp/manager.py

```diff
@@ -714,8 +716,8 @@ def get_validation_result(self, use_scratch_stage: bool):
 
     def get_events(
         self,
-        since_interval: str = "",
-        until_interval: str = "",
+        since: str | datetime | None = "",
+        until: str | datetime | None = "",
```

Comment:
Alright, I stared at this for too long, so at the risk of asking a dumb question, how do we get a datetime here from the `events` command?

Reply:
when we poll for events since the last query, we pass in the timestamp of the last result row from the previous call: https://github.com/snowflakedb/snowflake-cli/blob/a27e8fab746d1271a75e5b094ddc0a32f0c0f91d/src/snowflake/cli/plugins/nativeapp/manager.py#L804, which is automatically parsed as a `datatime` by the python connector

Reply:
python connector, that's the piece I was missing, thanks!